src/PaymentGateways/Gateways/Stripe/resources/js/StripeAccountPanel.tsx

```diff
@@ -0,0 +1,108 @@
+import {__} from '@wordpress/i18n';
+import {InspectorControls} from '@wordpress/block-editor';
+import {Notice, PanelBody, SelectControl, ToggleControl} from '@wordpress/components';
+import {createInterpolateElement} from '@wordpress/element';
+import {getFormBuilderWindowData} from '@givewp/form-builder/common/getWindowData';
+import './styles.scss';
+
+export default function StripeAccountPanel({
+    attributes,
+    setAttributes,
+}: {
+    attributes: StripeProps;
+    setAttributes: ({}) => void;
+}) {
+    const {gatewaysGlobalSettings, gatewaySettingsUrl} = getFormBuilderWindowData();
+    const stripeGlobalSettings: StripeGlobalSettingsProps = gatewaysGlobalSettings?.stripe_payment_element ?? {};
+
+    const textWithLinkToStripeSettings = (textTemplate: string) =>
+        createInterpolateElement(textTemplate, {
+            a: <a href={`${gatewaySettingsUrl}&section=stripe-settings&group=accounts`} />,
+        });
+
+    const hasGlobalDefault = stripeGlobalSettings.default;
+    const hasPerFormDefault = attributes.stripeAccountId;
+    const showGlobalDefaultNotice =
+        !hasGlobalDefault &&
+        (attributes.stripeUseGlobalDefault || (!attributes.stripeUseGlobalDefault && !hasPerFormDefault));
+
+    const useGlobalDefaultHelper = textWithLinkToStripeSettings(
```

Comment:
Just a heads up the prefix `use` is technically a reserved word in React for hooks.  I think if we had linting setup for react it would catch this from the rules of hooks (this is something we need to add in the future).

I would reccomend changing this function name to `shouldUseGlobalDefaultHelper` to avoid any confusion.

Reply:
Sure

Reply:
Resolved: https://github.com/impress-org/givewp/pull/6903/commits/3603f59670767ac18835e05beedab50185983736